### ISSUE TITLE
obsolete / double translations

### ISSUE THREAD

**Issue by maxxer** [zammad/zammad#2665]
Zammad 3.0.x

![immagine](https://user-images.githubusercontent.com/240201/61204815-87115c80-a6ee-11e9-8152-d25f59461200.png)

The two strings can probably be unified in one.

**Comment by dennykorsukewitz** [comment]
Hi @maxxer, 
thanks for report.

`Maximum size in MB of emails.` - correct / new

`Maximal size in MB of emails.` - wrong / old

`Maximal size in MB of emails.` is no longer available in the last version, but still available on the i18n server for older Zammad versions and also on your Zammad.

We are working on finding a solution to remove the old translations without deleting your own translations.

If the translation still bothers you, you can use this workaround.

```
rails console
Translation.find_by(source: 'Maximal size in MB of emails.').delete
```

**Comment by mgruner** [comment]
This will be fixed by the Weblate migration in Zammad 5.1. Translations added by the sync mechanism will be automatically cleaned up again if they are no longer needed and not changed by the user.
This does not, however, apply to translations of the time from before 5.1. These need to be manually removed by the user as described by @dennykorsukewitz, if required.
